packages/calypso-polyfills/README.md

```diff
@@ -0,0 +1,30 @@
+# Calypso Polyfills
+
+This package includes several configurations for required polyfills for Calypso, targeting three different environments: node, evergreen (newer browsers) and fallback (older browsers).
+
+## Features
+
+`calypso-polyfills` ties into the Calypso build process, to ensure that only the required polyfills for each build target are needed. Through the use of Babel's `preset-env` and `browserslist` configurations, we strip out any polyfills that are supported for all browsers that are supported by that build target.
+
+See the root directory's `package.json` for the list of supported browsers for each build target.
+
+## Usage
+
+In node, simply require the package:
+
+```js
+  require( '@automattic/calypso-polyfills' );
+```
+
+In the client, a similar naked import will include the polyfills (defaulting to the `fallback` set):
```

Comment:
Nit: "In a browser" is more precise wording than "In the client"

Reply:
Fixed.